### DISCUSSION TITLE
Dev help

### DISCUSSION THREAD

**Discussion by mediatech15** [OMOTE-Community/OMOTE-Firmware#42] [General]
What is the need for Dev help? I came across this on hackaday and read up on it. Being someone who has about 8 spare esps sitting around and always looking for a project.... I was wondering what help is needed as I am an openhab user looking for a generic remote that I can integrate and love open source solutions. I haven't bought a PCB setup yet. Saw the pcbway discussion.

Is this actively developed or a super side project. I would love to contribute when not working my day job.

**Comment by Dark1886** [comment]
@mediatech15 We are for sure looking for dev help here. We are currently working on a big overhaul of the software (and by WE, I mean @MatthewColvin). Feel free to contribute, but I would recommend chatting with @MatthewColvin  so you don't end up wasting time on something that already exists/will break with the software update.

**Comment by mediatech15** [comment]
Gotcha definitely not a fan of wasted work. I also have multiple 3d printers and was gonna look at some resin prints for the shell. Maybe make some button packs and inlays also in my spare time.

**Reply by Dark1886** [reply]
Great! We do have some people who have bought PCBs and are selling the spares. You may want to check that out before you order more boards.
